app/utils/print/AnswerRowConverter.scala

```diff
@@ -57,7 +57,7 @@ class AnswerRowConverter @Inject()() {
           HtmlFormat.escape(messages(s"$labelKey.checkYourAnswersLabel", name)),
           x match {
             case NA => HtmlFormat.escape(messages("individualBeneficiary.roleInCompany.checkYourAnswersLabel.na"))
-            case _ => HtmlFormat.escape(messages(x.toString))
+            case _ => HtmlFormat.escape(messages("individualBeneficiary.roleInCompany." + x.toString))
```

Comment:
Very minor but could this be
`HtmlFormat.escape(messages(s"individualBeneficiary.roleInCompany.${x.toString}"))`